indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -229,6 +229,7 @@ private static IncrementalIndex makeIncrementalIndex(
 
     OnheapIncrementalIndex newIndex = new OnheapIncrementalIndex(
         indexSchema,
+        true,
```

Comment:
hadoop batch ingestion has a config `ignoreInvalidRows` which should probably be used here instead.